components/StoryList/StyledStoryList.js

```diff
@@ -55,9 +55,9 @@ export const StoryListContainer = styled.ul`
 
 export const StoryListCard = styled.div`
   display: flex;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  /* border: 1px solid #ccc;
+  border-radius: 4px; */
+  /* box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); */
```

Comment:
If your code doesn't need these lines you can remove them. As far as I know we're not supposed to keep muted code.

Reply:
You are absolutely right! Will be removed